packages/aragon-cli/scripts/extract-roles.js

```diff
@@ -20,11 +20,16 @@ const aOSRoles = [
   { id: 'APP_MANAGER_ROLE', name: 'Manage DAO apps' },
 ]
```

Comment:
Maybe we should split this list into `kernelRoles` and `aclRoles` too (`CREATE_PERMISSIONS_ROLE` is for the ACL, `APP_MANAGER_ROLE` is for the kernel)